docs/concept/ai/ianvs.md

```diff
@@ -13,9 +13,9 @@ The distributed synergy AI benchmarking Ianvs aims to test the performance of di
 The scope of Ianvs includes
 - Providing end-to-end benchmark toolkits across devices, edge nodes and cloud nodes based on typical distributed-synergy AI paradigms and applications.
   - Tools to manage test environment. For example, it would be necessary to support the CRUD (Create, Read, Update and Delete) actions in test environments. Elements of such test environments include algorithm-wise and system-wise configuration.
-  - Tools to control test cases. Typical examples include paradigm templates, simulation tools, and hyper-parameter-based assistant tools.
+  - Tools to control test cases. Typical examples include paradigm templates, simulation tools, and hyper-parameter-based assistant tools. Hyper-parameters are settings that can affect how well your AI models learn from the data.
```

Comment:
cc @MooreZheng

Reply:
The explanation on hyperparameters looks fine to me.

Reply:
@anvithks  @edisonxiang